### PR TITLE
compilers: clspv: add -g

### PR DESCRIPTION
**Pull request by rjodinchr**
This commit will need to be merged only once `clspv` has been update with a newest version supporting this option.

### CODE REVIEW

File: lib/compilers/clspv.ts [hunk 1]
```diff
@@ -90,7 +90,7 @@ export class CLSPVCompiler extends BaseCompiler {
     override optionsForFilter(filters, outputFilename) {
         const sourceDir = path.dirname(outputFilename);
         const spvBinFilename = path.join(sourceDir, this.outputFilebase + '.spv');
```
Comment (dkm) [inline lib/compilers/clspv.ts:92]: Shouldn't this be factored out in `getOutputFilename` ?

**Review by dkm — COMMENTED** [review]
Also, I see some lib and binary settings. Is that needed/applicable for this compiler?

File: lib/compilers/clspv.ts (hunk 1 above)
Comment (rjodinchr) [reply lib/compilers/clspv.ts]: done

**Comment by rjodinchr** [comment]
> Also, I see some lib and binary settings. Is that needed/applicable for this compiler?

I guess you are talking about `prepareArguments`?
I started from `spirv.js`. That's why this function is there. I suppose it could be completely removed.
`clspv` has no use for lib and binary at the moment.

Can anyone confirm that in this case, this function should not be there?
Thanks

**Comment by rjodinchr** [comment]
Because `clspv` has a daily build, I expect it to be ready now.
I think this PR should be ready to be fully reviewed and then potentially merged.

**Review by RubenRBS — APPROVED** [review]

**Review by dkm — APPROVED** [review]

**Comment by rjodinchr** [comment]
This PR is ready to be merged: https://godbolt.org/z/W8jszxeoY

**Comment by dkm** [comment]
Do you want to rebase/squash before we merge your commits ?

**Comment by rjodinchr** [comment]
Everything can be squashed. I can't do it right now, but feel free to squash them yourself if you can.
Otherwise, I'll do it later today.

**Pull request merged**